docs/DatabaseChecks.md

```diff
@@ -36,4 +36,16 @@ If the config value is set to false the check will be skipped.
 
 ```json
 "CheckForOversizedIndexes": true
-```
\ No newline at end of file
+```
+
+## Fixed-Size Filegroups  
+Reports filegroups that are not set to auto-grow. You can whitelist fixed size filegroups by adding the name to the config array
+
+```json
+"ZeroAutoGrowthWhitelistFGs":[
```

Comment:
Is an empty array the way to turn this check on with no whitelist?

Reply:
This is [addressed in-line the check](https://github.com/taddison/SQLChecks/pull/5/files#diff-857031debf77af4a90920939e716d181R73). The default behavior is to execute globally. Removing the check entirely requires physically removing the pester or maintaining a whitelist of all filegroup names in the config. I considered allowing for a bypass value of "`ALL`", but testing for that felt like premature optimization for an edge case. Happy to put it in if you feel different, though 🤷‍♂️

This behavior when encountering `$null` input on the whitelist config value allows for easier testing as well as ad-hoc execution of the `Get-FixedSizeFileGroups` function after you run `.\RunAllChecks` (given that you've already imported the function). Since the `Get-FixedSizeFileGroups` ends by returning those filegroups that violate the pester, you can use immediately it to take action rather than just alerting that "`$Something` _is wrong._"

Reply:
Confirmed an empty array functions same as a missing config value in current build. 

```powershell
$config=('{"ServerInstance":"localhost","ZeroAutoGrowthWhitelistFGs":[]}' | ConvertFrom-Json)
<# OR #>
#$config=('{"ServerInstance":"localhost"}' | ConvertFrom-Json)

Get-FixedSizeFileGroups -ServerInstance localhost -WhitelistFilegroups $ZeroAutoGrowthWhitelistFGs
```

Reply:
This might be a case where some explicit guidelines are better - what I'm aiming for is that by default you get _no checks_, so the lack of a config or an empty config should be read as 'don't do this check'.